src/main/java/uk/gov/hmcts/darts/task/service/impl/AdminAutomatedTasksServiceImpl.java

```diff
@@ -103,6 +111,48 @@ public DetailedAutomatedTask updateAutomatedTask(Integer taskId, AutomatedTaskPa
             log.info("Batch size for {} updated to {}", automatedTask.getTaskName(), automatedTaskPatch.getBatchSize());
         }
 
+        //Arm Autoamted Task updates
```

Comment:
Not sure if you need this comment, but there's a typo in it

Reply:
Good spot updated